### PR TITLE
add support for COMPOSER_PROJECT_NAME env var

### PR DESCRIPTION
**Pull request by danielozano**
docker use docker-compose.yml parent directory as prefix for project containers, networks, etc. So if you have nested config on 2 projects, like for example: 

- project1/config/dockergento/docker-compose.yml
- project2/config/dockergento/docker-compose.yml

The containers get overriden, because they share the same project prefix: dockergento_

This PR will allow us to include a new property COMPOSER_PROJECT_NAME into our dockergento settings that will work as docker object's name prefix

### CODE REVIEW

**Comment by danielozano** [comment]
@ihor-sviziev what do you think? Would like to know your comments on this!

**Comment by ihor-sviziev** [comment]
@danielozano what is use case? In which case project directory will not be suitable?

**Comment by danielozano** [comment]
It happens when you've 2 projects with the docker-compose files inside a directory with the same name (imagine you're using cloud, and you already have docker-compose.yml in your root dir)

So for example imagine you've 2 projects with the following setup:
 - project1-root-dir/config/dockergento/
   - docker-compose.yml; docker-compose.dev.linux.yml; docker-compose.dev.mac.yml
 - project-2-root-dir/config/dockergento/
   - docker-compose.yml; docker-compose.dev.linux.yml; docker-compose.dev.mac.yml

And in both cases you've this in your properties file
```
DOCKER_COMPOSE_FILE="./config/dockergento/docker-compose.yml"
DOCKER_COMPOSE_FILE_LINUX="./config/dockergento/docker-compose.dev.linux.yml"
DOCKER_COMPOSE_FILE_MAC="./config/dockergento/docker-compose.dev.mac.yml"
```

When you do dockergento start, the container, volume (etc) names will be for bot projects:
dockergento_service
dockergento_appdata

So you've several workarounds:
 - name your services (project-nignx), also use container_name and change the name of all the volumes and networks for every project
 - use the docker-compose -p flag in order to add a "scope" to the docker objects
 - the same than -p flag, but with a temporal env var called COMPOSER_PROJECT_NAME

**Comment by ihor-sviziev** [comment]
@danielozano,
In which directory are you running `dockergento` commands? In which directory you have magento installed?

**Comment by danielozano** [comment]
@ihor-sviziev yes, but you could have dockergento installed inside a magento subdirectory (its supposed to work).

 With a normal setup, you have:
 - magento-root-dir -> run `dockergento` here
   - ./docker-compose.yml files
   - ./config/dockergento/ Dockerfiles, conf files, properties etc.

But dockergento is ready for having docker-compose files inside whatever subdirectory you need.

So it will search for your `properties` file, and read the properties: `DOCKER_COMPOSE_FILE, DOCKER_COMPOSE_FILE_LINUX, DOCKER_COMPOSE_FILE_MAC` which will be used to run docker-compose command.

Then if your docker-compose files aren't under root dir but inside a directory with the same name in both projects it will break because they share the same prefix.

Imagine this structure:
 - project-1-root-dir: Magento is installed here and I run `dockergento` here
   - ./config/dockergento/docker-compose.yml
 - project-2-root-dir: Magento is installed here and I run `dockergento` here
   - ./config/dockergento/docker-compose.yml

docker-compose will take the parent directory: dockergento as prefix (instead of projec-N-roo-dir)

File: console/commands/setup.sh [hunk 1]
```diff
@@ -174,11 +174,21 @@ if [ "${PHP_VERSION}" != "${DEFAULT_PHP_VERSION}" ]; then
     printf "${COLOR_RESET}\n"
 fi
 
+read -p "Docker compose project name: [${MAGENTO_DIR}] " ANSWER_PROJECT_NAME
+PROJECT_NAME=${ANSWER_PROJECT_NAME-}
```
Comment (ihor-sviziev) [inline console/commands/setup.sh]: I think it should be shown that it's  optional

**Comment by ihor-sviziev** [comment]
@danielozano in this case it make sense

File: console/commands/setup.sh (hunk 1 above)
Comment (danielozano) [reply console/commands/setup.sh]: Any idea on how to show that its optional? Any standarised sentece or something? (some markup)

Comment (ihor-sviziev) [reply console/commands/setup.sh]: Maybe “Docker compose project name (optional):”?

Comment (danielozano) [reply console/commands/setup.sh]: Ok nice! I've updated it!

**Comment by danielozano** [comment]
Nice, thanks a lot for your help and your time @ihor-sviziev!

**Pull request merged**